### DISCUSSION TITLE
제페토 월드에서 다른 제페토 월드 링크로 연결시키는 것이 가능할까요?

### DISCUSSION THREAD

**Discussion by DoyoonAhn** [naverz/zepeto-studio-kor#203] [5. Q&A]
외부 URL은 승인이 안되는 것으로 알고 있는데 제페토 내에서 월드 이동은 가능한가요?

**Comment by hyeonjin-jo** [comment]
안녕하세요.

현재는 해당 기능을 제공하지 않고 있습니다.

이후 버전에서는 논의될 수도 있는 사항이지만 정확한 업데이트 시점을 공유해 드릴 수는 없는 점 양해 부탁드립니다.
